linq4j/src/test/java/org/apache/calcite/linq4j/test/InlinerTest.java

```diff
@@ -152,6 +154,25 @@ void checkAssignInConditionOptimizedOut(int modifiers, String s) {
     assertThat(Expressions.toString(builder.toBlock()), is(s));
   }
 
+  @Test void testInlineTernaryNonOptimized() {
```

Comment:
I think it's customary to add a Javadoc comment indicating the issue which is being solved. Please follow the format of the many examples existing, the tools are very particular about the format, including the final period.

Reply:
> I think it's customary to add a Javadoc comment indicating the issue which is being solved. Please follow the format of the many examples existing, the tools are very particular about the format, including the final period.

Thanks for your suggestion, I have added Javadoc comment on the test case.